externs/browser/w3c_indexeddb.js

```diff
@@ -386,15 +445,10 @@ IDBDatabase.prototype.createObjectStore =
 IDBDatabase.prototype.deleteObjectStore = function(name) {};
 
 /**
- * @param {string} version The new version of the database.
- * @return {!IDBRequest} The IDBRequest object.
- */
-IDBDatabase.prototype.setVersion = function(version) {};
-
-/**
- * @param {string|Array<string>} storeNames The stores to open in this
- *     transaction.
- * @param {(number|string)=} mode The mode for opening the object stores.
+ * TODO: Change mode param to !IDBTransactionMode once CC understands string literal types.
```

Comment:
Please don't refer to closure-compiler as "CC" or "GCC". Those abbreviations have a long history of referring to the C language compiler and its GNU implementation.